docs/docs/rest-api/public/api/v2/apps.raml

```diff
@@ -143,7 +143,9 @@
   patch:
     description:
       Change multiple existing applications by applying a patch.
-      All instances of these applications get replaced by the new version.
+      Instances of these applications will be started or restarted according to the ususal logic to meet the
+      requirements of the new app definitions. Usually, restart is required if the application configuration was changed
```

Comment:
Most of the time instead of usually here? So that we don't repeat the same word...